apps/condo/pages/contact/index.tsx

```diff
@@ -162,6 +204,8 @@ const ContactTableContent: React.FC<ContactPageContentProps> = (props) => {
                                             rowValidator={contactValidator}
                                             objectCreator={contactCreator}
                                             domainName='contact'
+                                            exampleModalAdditionalContent={IsVerifiedCheckbox}
```

Comment:
Why this prop called `exampleModalAdditionalContent`? I suppose it is not an example content

Reply:
because it is a additional content in a import example modal (first step of import modal – import example)